W-11722162-email-verification (#235)
mulesoft/docs-access-management: modules/ROOT/pages/managing-your-account.adoc

Connected apps and clients are exempt from reauthentication. If you have a script that makes an API call to change your email address, it might need to accommodate reauthentication.

=== Verify Your Email Address

> If you have not verified your email address that you use to log in to Anypoint Platform, a prompt appears to verify your email when you log in.

Author: ```suggestion
As of October 10, 2022, non-SSO users must verify their Anypoint Platform email addresses. If you have not verified your email address, an email verification request prompt appears each time you log in.
```

You can verify your email using the login prompt, or you can verify your email in your user profile.

> If you are an SSO user, you cannot verify your email address through Anypoint Platform.

Reviewer: to make it clear, the prompt won't appear for SSO users.

Author: ```suggestion
SSO users cannot verify their email addresses through Anypoint Platform and do not receive verification prompts.
```

include::partial$include-nav-steps-to-profile.adoc[]

. Click the *Contact Info* tab.
. In the *Email* section, click *Verify Email*. +

> A window appears, prompting you to enter a verification code that you received at your Anypoint Platform account's email address.

Reviewer 2: Instead of "received", should we say "will receive"? Sometimes it may take a minute to get the email with the verification code. The user may get confused as they will look for the code right after clicking on "Verify Email".

Author: No; we do not use future tense if we can help it. I think we should note that it might take the email a bit to arrive.

Author: ```suggestion
A window appears, prompting you to enter a verification code that you received at your Anypoint Platform account's email address. The verification email may take a few minutes to arrive. If you do not see the email in your inbox after several minutes, check your Spam folder.
```

Reviewer 2: That also works.

+
. Enter the verification code.
. Click *Confirm*. +
Your email address is verified. 

== Change Your Account Email Address

You can change the email address that you use to log in to your Anypoint Platform account.

[NOTE]
SSO users cannot change their account email address through Anypoint Platform.

include::partial$include-nav-steps-to-profile.adoc[]

. Click the *Contact Info* tab.
. In the *Email* row, click the pencil icon.
. Enter your new email address.
. Click *Save Changes*.
. If you have been logged in to Anypoint Platform for more than 30 minutes, reenter your password to reauthenticate. +
A window appears, prompting you to enter a verification code that you received at the new email address you entered.
+
. Enter the verification code.
. Click *Confirm*. +

> The email address you entered is now the email address you use to log in to Anypoint Platform.

Reviewer: This isn't technically true as they use the username to login

Author: ```suggestion
The email address you entered is now your primary email address for Anypoint Platform.
```
